src/libaktualizr/package_manager/packagemanagerfake.cc

```diff
@@ -110,5 +110,10 @@ bool PackageManagerFake::fetchTarget(const Uptane::Target& target, Uptane::Fetch
     return false;
   }
 
+  if (target.IsOstree()) {
```

Comment:
I would probably prefer something like `if (target.type_ != this->type_)`, but that should be also good for now.

Reply:
Me too, but for now we don't expose package manager types... unless I've forgotten something?

Reply:
No, we don't. I just was thinking that we might want to do something like this in the future,

Reply:
PackageManagerInterface::name() is kind of a package manager type :)

Reply:
@mike-sul True, but it still doesn't have 1:1 match with update type.

Reply:
Yeah, I know, maybe adding something like package_manager->isTargetSupported(const Target&)  (or logicalEcu->isTargetSupported()) so the pack manager implementation decides whether it can handle the given target or not would be the best here.